services/policies/pkg/config/config.go

```diff
@@ -0,0 +1,58 @@
+package config
+
+import (
+	"context"
+	"github.com/owncloud/ocis/v2/ocis-pkg/shared"
+	"time"
+)
+
+// Config combines all available configuration parts.
+type Config struct {
+	Commons           *shared.Commons `yaml:"-"` // don't use this directly as configuration for a service
+	GRPC              GRPC            `yaml:"grpc"`
+	Service           Service         `yaml:"-"`
+	TokenManager      *TokenManager   `yaml:"token_manager"`
+	Events            Events          `yaml:"events"`
+	Reva              *shared.Reva    `yaml:"reva"`
+	MachineAuthAPIKey string          `yaml:"machine_auth_api_key" env:"OCIS_MACHINE_AUTH_API_KEY;POLICIES_MACHINE_AUTH_API_KEY" desc:"Machine auth API key used to validate internal requests necessary for the access to resources from other services."`
+	Context           context.Context `yaml:"-"`
+	Engines           Engines         `yaml:"engines"`
+}
+
+// Service defines the available service configuration.
+type Service struct {
+	Name string `yaml:"-"`
+}
+
+// GRPC defines the available grpc configuration.
+type GRPC struct {
+	Addr      string `yaml:"addr" env:"POLICIES_GRPC_ADDR" desc:"The bind address of the GRPC service."`
+	Namespace string `yaml:"-"`
+}
+
+// TokenManager is the config for using the reva token manager
+type TokenManager struct {
+	JWTSecret string `yaml:"jwt_secret" env:"OCIS_JWT_SECRET;POLICIES_JWT_SECRET" desc:"The secret to mint and validate jwt tokens."`
+}
+
+// Engines wraps the available policy engine configurations.
+type Engines struct {
+	Timeout time.Duration `yaml:"timeout"`
+	OPA     OPA           `yaml:"opa"`
+}
+
+// OPA configures the opa policy engine.
+type OPA struct {
+	Enabled  bool     `yaml:"enabled" env:"POLICIES_ENGINES_OPA_ENABLED" desc:"Activates OPA policy engine."`
+	Query    string   `yaml:"query" env:"POLICIES_ENGINES_OPA_QUERY" desc:"Sets the OPA Rego query."`
```

Comment:
This sounds that you define a query (rule) here and only one.
As we are talking about policies (plural), I expect more than one rules to be possible. And in such a case, how will we define multiple queries? why not specifying a file where all the rules are located? Maybe I have a wrong view.

Reply:
this is fixed, the query is now part of the grpc request